{
  "id": "uno-ooo-parser",
  "version": "0.1.22",
  "libre_office_ver": "7.2",
  "name": "InvalidTextContentException",
  "type": "exception",
  "namespace": "com.sun.star.text",
  "parser_args": {
    "sort": false,
    "long_names": true,
    "remove_parent_inherited": true
  },
  "writer_args": {
    "include_desc": true
  },
  "data": {
    "allow_db": true,
    "from_imports": [
      [
        "..uno.exception",
        "Exception",
        "Exception_85530a09"
      ]
    ],
    "from_imports_typing": [
      [
        ".x_text_content",
        "XTextContent",
        "XTextContent_b16e0ba5"
      ]
    ],
    "extends_map": {
      "com.sun.star.uno.Exception": "Exception_85530a09"
    },
    "quote": [
      "XTextContent_b16e0ba5"
    ],
    "typings": [],
    "requires_typing": true,
    "full_imports": {
      "general": [
        "com.sun.star.uno.Exception"
      ],
      "typing": [
        "com.sun.star.text.XTextContent"
      ]
    },
    "name": "InvalidTextContentException",
    "imports": [],
    "namespace": "com.sun.star.text",
    "extends": [
      "com.sun.star.uno.Exception"
    ],
    "desc": [
      "is thrown whenever a method gets a TextContent as an actual argument when the text content cannot be used for that operation."
    ],
    "url": "https://api.libreoffice.org/docs/idl/ref/exceptioncom_1_1sun_1_1star_1_1text_1_1InvalidTextContentException.html",
    "items": {
      "properties": [
        {
          "name": "TextContent",
          "returns": "XTextContent_b16e0ba5",
          "desc": [
            "contains the interface of the text content that caused the exception."
          ],
          "raises_get": "",
          "raises_set": ""
        }
      ]
    }
  }
}